packages/trusted-contribution/src/trusted-contribution.ts

```diff
@@ -105,9 +102,37 @@ export = (app: Probot) => {
       if (remoteConfiguration.disabled) {
         return;
       }
+
+      // Check if repository is configured for OwlBot, by fetching .github/.OwlBot.yaml,
+      // which exists in all OwlBot-enabled repositories:
+      let hasOwlBotConfig = false;
+      try {
+        await context.octokit.rest.repos.getContent({
+          owner,
+          repo,
+          path: OWLBOT_CONFIG_PATH,
+        });
+        hasOwlBotConfig = true;
+      } catch (_err) {
+        const err = _err as {code: number};
```

Comment:
Is it safe to catch this error in all cases? Should we throw it in some cases?

Reply:
That's a good question, it might be worth throwing on a non-404 (I think this should happen almost never).